### ISSUE TITLE
Treeshaking breaks sprite build

### ISSUE THREAD

**Issue by AndyOGo** [mrsum/webpack-svgstore-plugin#120]
Using [`rollup-loader`](https://www.npmjs.com/package/rollup-loader) to utilize treeshaking removes all function marks like `var __svg__`, `var __sprite__`, etc.

Hence no sprites get generated.

**Comment by mrsum** [comment]
Is it [WIP] currently, right?

**Comment by AndyOGo** [comment]
@mrsum 
Yeah it is WIP, actuallyI'm not certain [if it got abandoned](https://github.com/egoist/rollup-loader/issues/1) in favour of webpack 2.

Anyway function marks will fail with treeshaking approach.

**Comment by mrsum** [comment]
@AndyOGo Okay, i will fix it ASAP.
